src/v2/view-builder/views/TerminalView.js

```diff
@@ -121,31 +121,16 @@ const Body = BaseForm.extend({
       messagesObjs.value.push({ message: loc('oie.return.to.original.tab', 'login')});
     } else if (this.options.appState.containsMessageWithI18nKey('tooManyRequests')) {
       description = loc('oie.tooManyRequests', 'login');
+    } else if (this.options.appState.containsMessageWithI18nKey(RETURN_LINK_EXPIRED_KEY)) {
+      messagesObjs.value[0].class = 'ERROR';
```

Comment:
how do you know it is the first message? Before we were adding an error callout if the message had `RETURN_LINK_EXPIRED_KEY` as the key, but we would show other messages if they were available.

I think we need to know the index of this error and change that specific one. Most of our errors today are an array of 1 element, but I don't think that is part of the contract, so we shouldn't assume it. At the end, if `messages` is an array, it can contain any number of items

Reply:
I would create a backend jira for this.. The flow expects this to be `ERROR` but api is returning this error as `INFO`.. This needs to be fixed on backend side, so that i can remove this one off hack from widget.. 

https://github.com/okta/okta-core/blob/eed2411d033788729198661374245ae3d9562728/components/platformx/idx/api/src/main/java/com/okta/api/mediation/services/auth/TerminalStateTokenHelper.java#L121

https://oktainc.atlassian.net/browse/OKTA-443718